Xamarin.Forms.Core/IListViewController.cs

```diff
@@ -15,6 +15,8 @@ public interface IListViewController : IViewController
 		string GetDisplayTextFromGroup(object cell);
 		void NotifyRowTapped(int index, int inGroupIndex, Cell cell);
 		void NotifyRowTapped(int index, Cell cell);
+		void NotifyRowTapped(int index, int inGroupIndex, Cell cell, bool isContextMenuRequested);
```

Comment:
Adding to a public interface is also a breaking change. We'd need to add a new interface.

Reply:
ListView : IListViewController, IListViewController2

Public interface IListViewController2
{
void NotifyRowTapped(int index, int inGroupIndex, Cell cell, bool isContextMenuRequested);
}

Something like this?